docs/cloud/satellites/byoc/byoc.md

```diff
@@ -0,0 +1,70 @@
+# Bring Your Own Cloud
+
+"Bring Your Own Cloud" (BYOC) satellites **Experimental** are a hybrid between [self hosted](../self-hosted.md) and [earthly-managed](../../satellites.md) satellites. These satellites are managed by Earthly; just like our managed offering, but within your infrastructure. This blends the ease-of-use of managed satellites with the security benefits that self hosting can bring.
+
+Here is how Earthly-managed satellites compare to BYOC: 
+
+| Similarities                                                                               | Differences                                                           |
+|--------------------------------------------------------------------------------------------|-----------------------------------------------------------------------|
+| ✅ The ability to use `earthly satellite` commands to provision and decommission satellites | ❌ It is up to you to get traffic to and from the satellite            |
+| ✅ Earthly-provided monitoring and reliability                                              | ❌ Earthly cannot access the machines for debugging or troubleshooting |
+| ✅ Automatic updates                                                                        |                                                                       |
+| ✅ Automatic sleep/wake                                                                     |                                                                       |
+
+Here is how Self-hosted satellites compare to BYOC:
+
+| Similarities                                                     | Differences                                                                            |
+|------------------------------------------------------------------|----------------------------------------------------------------------------------------|
+| ✅ The satellite lives in your infrastructure, next to your tools | ❌ You cannot manually provision satellites                                             |
+| ✅ You pay for the usage the satellite incurs                     | ❌ You must allow Earthly access to at least some portion of an AWS account you control |
+|                                                                  | ❌ You pay the cloud provider, not Earthly, for the compute usage                       |
```

Comment:
The most important difference I would add here is that we manage the up-time of satellites, whereas with self-hosted, they manage the up-time.